functions/Measure-DbaDiskSpaceRequirement.ps1

```diff
@@ -70,186 +76,224 @@ Function Measure-DbaDiskSpaceRequirement {
+
+        if ($destDb = Get-DbaDatabase -SqlInstance $destServer -Database $DestinationDatabase -SqlCredential $DestinationSqlCredential) {
+            $destFiles = @($destDb.FileGroups.Files | Select-Object Name, FileName, Size, @{n='Type'; e= {'Data'}})
+            $destFiles += @($destDb.LogFiles        | Select-Object Name, FileName, Size, @{n='Type'; e= {'Log'}})
+            $computerName = $destDb.ComputerName
+        }
+        else {
+            Write-Message -Level Verbose -Message "Database [$DestinationDatabase] does not exist on Destination Instance $Destination."
+            $computerName = $destServer.NetName
         }
 
-        foreach($File1 in $DataFiles1) {
-            foreach($File2 in $DataFiles2) {
-                if($found = ($File1.Name -eq $File2.Name)) { # Files found on both sides
-                    $Detail += @([PSCustomObject]@{
-                        Source = $Source
-                        Destination = $Destination
-                        DatabaseName1 = $DB1.Name
-                        DatabaseName2 = $DB2.Name
-                        Name1 = $File1.Name
-                        Name2 = $File2.Name
-                        FilePath1 = $File1.FileName
-                        FilePath2 = $File2.FileName
-                        SizeKB1 = $File1.Size
-                        SizeKB2 = $File2.Size * -1
-                        DiffKB = $File1.Size - $File2.Size
-                        ComputerName = $ComputerName
-                        MountPoint = Get-MountPointFromPath -Path $File2.Filename -ComputerName $ComputerName -Credential $Credential
-                    })
+        foreach ($sourceFile in $sourceFiles) {
+            foreach ($destFile in $destFiles) {
+                if ($found = ($sourceFile.Name -eq $destFile.Name)) {
+                    # Files found on both sides
+                    $details += @([PSCustomObject]@{
+                            SourceComputerName      = $sourceServer.NetName
+                            SourceInstance          = $sourceServer.ServiceName
+                            SourceSqlInstance       = $sourceServer.DomainInstanceName
+                            DestinationComputerName = $destServer.NetName
+                            DestinationInstance     = $destServer.ServiceName
+                            DestinationSqlInstance  = $destServer.DomainInstanceName
+                            SourceDatabase          = $sourceDb.Name
+                            DestinationDatabase     = $destDb.Name
+                            SourceLogicalName       = $sourceFile.Name
+                            DestinationLogicalName  = $destFile.Name
+                            SourceFileName          = $sourceFile.FileName
+                            DestinationFileName     = $destFile.FileName
+                            SourceFileSizeKB        = $sourceFile.Size
```

Comment:
Can we use [dbasize] here?

Reply:
And anywhere else relevant. may require changing to bytes

Reply:
This is just returning the raw value, which is in KB. I can see how the DB space command handles it and just match that.